js/style/style.js

```diff
@@ -18,12 +18,13 @@ var QueryFeatures = require('../source/query_features');
 var SourceCache = require('../source/source_cache');
 var styleSpec = require('./style_spec');
 var StyleFunction = require('./style_function');
+var getWorkerPool = require('../global_worker_pool');
```

Comment:
We can, but it makes [this code](https://github.com/mapbox/mapbox-gl-js/pull/2952/files#diff-bd78950980d7affe72dbe3366d8f03f2R9) problematic: it means that there's no chance for client code to set `mapboxgl.WorkerSource.WORKER_COUNT` before the global pool is constructed.  We could work around this by having the worker pool wait to look up `WORKER_COUNT` until it's actually creating workers during the first `acquire` call.